.github/workflows/run-client.yml

```diff
@@ -0,0 +1,67 @@
+name: run client
+on:
+  push:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  pipeline-user-scenario:
+    name: Build on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node_version: ['12']
+        os: [ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node_version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node_version }}
+    - name: building
+      run: |
+        npm install ts-node -g
```

Comment:
Why do we need to install ts-node?

Reply:
It will throw err: 'ts-node not found' if not install here :(

Reply:
The 'ts-node' in dir 'cli/node_modules/.bin/ts-node' can not be found by the cli client, so I install it here :(

Reply:
`npm run build` will compile all the ts files, and generates to its dist directory, maybe you should execute there.

Reply:
'ts-node' has installed in /cli/node_modules but not be found.
The file after build in dist dir but can not find 'ts-node' bin file :(

Reply:
What I meant is to run cli without ts-node, because we have compiled it to JavaScript.

Reply:
OO